pkg/await/core_service_test.go

```diff
@@ -242,15 +203,6 @@ func Test_Core_Service_Read(t *testing.T) {
 		version           serverVersion
 		expectedSubErrors []string
 	}{
-		{
-			description:  "Read should fail if Service does not target any Pods",
```

Comment:
I feel like there is still a case to test this for Service creation success or failure, even when there are no Pods. thoughts?